packages/atclient/include/atclient/atclient.h

```diff
@@ -56,4 +61,10 @@ int atclient_get(atclient *ctx, const char *key, char *value, const unsigned lon
 int atclient_delete(atclient *ctx, const char *key);
 void atclient_free(atclient *ctx);
 
+int atclient_get_encryption_key_shared_by_me(atclient *ctx, atclient_atsign *recipient, char *enc_key_shared_by_me);
+int atclient_get_encryption_key_shared_by_other(atclient *ctx, atclient_atsign *recipient, char *enc_key_shared_by_other);
```

Comment:
Missing documentation on header functions.

Important to mention which arguments are required, optional, also important to mention that *ctx is assumed to be initialized via the atclient_init function.